gh.go

```diff
@@ -34,6 +34,16 @@ func Exec(args ...string) (stdOut, stdErr bytes.Buffer, err error) {
 	return run(path, nil, args...)
 }
 
+// Exec gh command interactively with provided arguments.
+func ExecInteractive(args ...string) (err error) {
```

Comment:
Since this is meant to spawn a potentially longer-running process, this function could accept a `context.Context` instance so that the caller has an option to kill the gh process when it needs to.